src/switch_core_media.c

```diff
@@ -5334,13 +5332,15 @@ static void check_stream_changes(switch_core_session_t *session, const char *r_s
 			if (switch_channel_test_flag(other_session->channel, CF_AWAITING_STREAM_CHANGE)) {
 				uint8_t proceed = 1;
 				const char *sdp_in, *other_ep;
+				uint8_t res = 0;
 
 				if ((other_ep = switch_channel_get_variable(session->channel, "ep_codec_string"))) {
 					switch_channel_set_variable(other_session->channel, "codec_string", other_ep);
 				}
 
 				sdp_in = switch_channel_get_variable(other_session->channel, SWITCH_R_SDP_VARIABLE);
-				switch_core_media_negotiate_sdp(other_session, sdp_in, &proceed, SDP_TYPE_REQUEST);
+				res = switch_core_media_negotiate_sdp(other_session, sdp_in, &proceed, SDP_TYPE_REQUEST);
```

Comment:
Should this be replace with `SDP_OFFER/SDP_ANSWER`

Reply:
hmm.. i don't see `SDP_OFFER/SDP_ANSWER` being defined anywhere in 1.10.11, or am I missing something?